store/blocks/search.jsonc

```diff
@@ -226,5 +226,13 @@
 
   "gallery": {
     "blocks": ["product-summary.shelf"]
+  },
+
+  "back-to-top-button":{
+    "props":{
+      "displayThreshold": 800,
+      "display":"caret-icon"
+    }
```

Comment:
Block definitions are shared across files, so you only need to define a block once.

Reply:
We get all JSON files and compile it to one big JSON.